### PR TITLE
adds updated schema.db

### PR DESCRIPTION
**Pull request by amprokop**
Checks in schema— my oversight that it wasn't checked in, I think. This shouldn't have caused any problems though, since this file is auto-generated.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -107,6 +107,7 @@
     t.string   "_initial_ssoc_required"
     t.datetime "created_at",                                    null: false
     t.datetime "updated_at",                                    null: false
+    t.string   "certifying_official_title_specify_other"
```
Comment (joofsh) [inline db/schema.rb]: Can you add more explanation for this? I don't see a migration that adds this new column. It's not being referenced anywhere in the code

Comment (amprokop) [inline db/schema.rb]: arg. sorry this is wrong. it's covered in https://github.com/department-of-veterans-affairs/caseflow/pull/545 which i thought was merged but isn't

Comment (amprokop) [inline db/schema.rb]: sorry to waste your time, this doesn't need to be merged my bad

**Comment by amprokop** [comment]
closing bc https://github.com/department-of-veterans-affairs/caseflow/pull/545 will do this

**Pull request closed without merging**